cli/commands/run/server/agent.controller.spec.ts

```diff
@@ -11,6 +11,7 @@ describe("AgentController", () => {
   const mockCallback = jest.fn()
   const mockFinding = { some: 'finding' }
   const systemTime = new Date()
+  const consoleSpy = jest.spyOn(console, 'log');
```

Comment:
do we need this since we're just throwing errors (and not logging)?

Reply:
Updated